docs/content/migration.md

```diff
@@ -0,0 +1,535 @@
+| $mdIcon          |   [Docs][131]|   [Docs][132]| `svgIcon`                                              |
+| $mdIconProvider  |   [Docs][133]|   [Docs][134]| `MatIconRegistry`                                      |
+| $mdInkRipple     |   [Docs][58] |   [Docs][19] | `matRippleTrigger`                                     |
+| $mdInkRippleProvider|[Docs][135]|   [Docs][136]| `MAT_RIPPLE_GLOBAL_OPTIONS`                            |
+| $mdInteraction   |   [Docs][137]|   N/A        |                                                        |
+| $mdLiveAnnouncer |   [Docs][111]|   [Docs][112]| CDK `LiveAnnouncer`                                    |
+| $mdMedia         |   [Docs][137]|   [Docs][138]| CDK `BreakpointObserver.isMatched()` or Angular Layout's [MediaObserver Service][119]|
+| $mdPanel         |   [Docs][109]|   [Docs][110]| CDK `Overlay`                                          |
+| $mdPanelProvider |   [Docs][139]|   N/A        |                                                        |
+| $mdProgressCircularProvider|[Docs][140]|N/A    |                                                        |
+| $mdSidenav       |   [Docs][141]|   N/A        |                                                        |
+| $mdSticky        |   [Docs][142]|   N/A        |                                                        |
+| $mdTheming       |   [Docs][143]|   N/A        | [Sass mixins][145] support static theming. No support for dynamic theming.|
+| $mdThemingProvider|  [Docs][144]|   N/A        | [Sass mixins][146] support static, custom theme creation. Use Angular's [Meta Service][147] for browser coloring features.|
+| $mdToast         |   [Docs][63] |   [Docs][21] | `MatSnackBar`                                          |
+
+### <a name="new-components"></a> New Components
+These are new components found in Angular Material and the Angular CDK that do not exist in
+AngularJS Material.
+
+The `mat-` and `cdk-` prefixes have been omitted to make this table more readable.
+
+| Directive        |     Docs     | Notes                                                  |
+|------------------|--------------|--------------------------------------------------------|
+| badge            |   [Docs][37] |                                                        |
+| button-toggle    |   [Docs][15] |                                                        |
+| table            |   [Docs][28] | Non-themed CDK and Material versions available.        |
+| drag-drop        |   [Docs][39] | CDK                                                    |
+| expansion-panel  |   [Docs][32] |                                                        |
+| paginator        |   [Docs][29] |                                                        |
+| sort-header      |   [Docs][30] |                                                        |
+| stepper          |   [Docs][33] | Non-themed CDK and Material versions available.        |
```

Comment:
"Non-themed" -> "Unstyled"

Reply:
OK, will update. I went with that first, but my spell checker complained.